app/stacks/global/front-door/outputs.tf

```diff
@@ -0,0 +1,9 @@
+output "frontend_endpoint_mappings" {
+  description = "A map of maps containing configuration options for each frontend endpoint"
+  value       = local.frontend_endpoint_mappings
+}
+
+output "frontend_endpoints" {

```

Comment:
Can we rename to "frontend_endpoint_ids"?

Reply:
Same with the variable in the https stack.

Reply:
Actually have changed my mind, because then you'd have to create another variable just to extract the ids! Which is more pain than needed!